WizardEngine/Entity.cpp

```diff
@@ -1,10 +1,24 @@
 #include "Entity.h"
 #include "Component.h"
 
-Entity::Entity(Mesh * a_pMesh, Material* a_pMaterial, DirectX::XMFLOAT4X4 a_mWorld, DirectX::XMFLOAT3 a_vPos, DirectX::XMFLOAT3 a_vRotation, DirectX::XMFLOAT3 a_vScale) : m_pMesh(a_pMesh), m_mWorld(a_mWorld),
-m_vPos(a_vPos), m_vRotation(a_vRotation), m_vScale(a_vScale), m_pMaterial(a_pMaterial)
+using namespace DirectX;
+
+Entity::Entity(Mesh * mesh, Material* material)
```

Comment:
I changed the constructor because the other one required a lot of params up front. Having default values for position, scale and rotation is nice.